charts/kusion/README.md.gotmpl

```diff
@@ -79,8 +85,8 @@ helm install kusion-release kusionstack/kusion \
 --set server.replicas=3 \
 --set mysql.enabled=true \
 --set mysql.database=kusion \
---set kubeconfig.kubeConfigs.kubeconfig0="$KUBECONFIG_CONTENT1" \
---set kubeconfig.kubeConfigs.kubeconfig1="$KUBECONFIG_CONTENT2"
+--set kubeconfig.kubeConfigs.kubeconfig-0="$KUBECONFIG_CONTENT1" \
```

Comment:
Using hyphens in the kubeconfig keys (`kubeconfig-0` and `kubeconfig-1`) might cause issues with some Helm templating or YAML parsers. Consider using underscores (`kubeconfig_0` and `kubeconfig_1`) instead for better compatibility.